s3config.yaml

```diff
@@ -69,6 +69,7 @@ S3_SERVER_CONFIG:
    S3_REDIS_SERVER_PORT: 6379                           # In case if redis is used for kvs contains redis server port
    S3_SERVER_MOTR_ETIMEDOUT_MAX_THRESHOLD: 100          # Number of ETIMEDOUT errors per monitoring window before s3server restart
    S3_SERVER_MOTR_ETIMEDOUT_WINDOW_SEC: 1               # Monitoring window for motr ETIMEDOUT errors in seconds
+   S3_SERVER_ENABLE_ADDB_DUMP: true                     # If set to true, then addb dump will be collected
```

Comment:
On Dev environment, do we need it to be enabled by default ?

Reply:
It was enabled in dev environment (in startup script, we were passing --addb option) previously as well. SO kept it same.

Should we set it to False for dev env ?

Reply:
Ok, let it be like that. 
Second thought says that we may be able to catch some issue in Dev itself.